tests/common/util/LoggerFixtures.hpp

```diff
@@ -19,56 +19,25 @@
 
 #pragma once
 
-#include "util/log/Logger.hpp"
+#include "util/StringBuffer.hpp"
 
 #include <gtest/gtest.h>
 
-#include <algorithm>
-#include <mutex>
 #include <ostream>
-#include <sstream>
 #include <string>
 
 /**
- * @brief Fixture with util::Logger support.
+ * @brief Fixture for LogService.
```

Comment:
Maybe something like `A fixture for testing LogService and Logger`?

Reply:
Fixture just provides context for some test(s). Does not have to be "for a class" at all :) I just don't think this brief describes this fixture's intention

Reply:
Thanks, changed